lib/tasks/webpack_try_compile.rake

```diff
@@ -2,9 +2,9 @@ namespace :webpack do
   desc 'Try to compile webpack assets for integration tests, fails only with a warning'
   task :try_compile do
     begin
-      Rake::Task['webpack:compile'].invoke
+      Rake::Task['webpacker:compile'].invoke
     rescue => e
-      puts "WARNING: `rake webpack:compile` failed to run. This is only important if running integration tests. (cause: #{e})"
+      puts "WARNING: `rake webpacker:compile` failed to run. This is only important if running integration tests. (cause: #{e})"
```

Comment:
does it make sense to support also the old task name, so packaging/scripting do not need to update for this change?

Reply:
It does make sense, what about supporting the old task names with a deprecated warning?
We can keep it for a few releases and then stop to support it.

I think we need a global approach about how to deprecate methods.